recipes/config.rb

```diff
@@ -40,9 +40,18 @@
 bin = ::File.join(node['qubit_bamboo']['home'], 'bamboo')
 flags = node['qubit_bamboo']['flags'].sort.map { |k, v| " -#{k}=#{v}" }.join ' '
 syslog = node['qubit_bamboo']['syslog'] ? '2>&1 | logger -p user.info -t bamboo' : ''
-bamboo_command = %("#{bin} #{flags} #{syslog}")
+
+node.default['qubit_bamboo']['poise_service']['options'] = {
```

Comment:
because otherwise, it will compute an incomplete command.
If you have a better solution, i'm all ears :)